packages/contracts-bedrock/test/DeployImplementations.t.sol

```diff
@@ -30,6 +31,7 @@ import {
 contract DeployImplementationsInput_Test is Test {
     DeployImplementationsInput dii;
 
+    bytes32 salt = 0x1000000000000000000000000000000000000000000000000000000000000001;
```

Comment:
nit, let's default to the zero salt for consistency/simplicity unless we need to override it
```suggestion
    bytes32 salt = btyes32(0);
```

Reply:
Yea, I didn't want to use zero here because I wasn't sure if it defaults to zero if it doesn't exist. I wanted to make sure it was being read.